qfieldcloud_sdk/sdk.py

```diff
@@ -85,29 +153,75 @@ class OrganizationMemberModel(TypedDict):
 
 
 class Pagination:
+    """The Pagination class allows for controlling and managing pagination of results within the QFieldCloud SDK.
+
+    Attributes:
+        limit (int | None): The maximum number of items to return.
+        offset (int | None): The starting point from which to return items.
+    """
+
     limit = None
     offset = None
 
     def __init__(
         self, limit: Optional[int] = None, offset: Optional[int] = None
     ) -> None:
+        """Initializes the pagination settings.
+
+        Args:
+            limit (int | None, optional): The maximum number of items to return. Defaults to None.
+            offset (int | None, optional): The starting point from which to return items. Defaults to None.
+        """
         self.limit = limit
         self.offset = offset
 
     @property
     def is_empty(self):
+        """Checks if both limit and offset are None, indicating no pagination settings.
```

Comment:
```suggestion
        """Whether both limit and offset are None, indicating no pagination settings.
```
Note this is not a function, but a `@property`, which in python is  a getter that behaves as an attribute.